### PR TITLE
streaming frames: add a simple decode-labels-only mode

### PR DESCRIPTION
**Pull request by gabor**
work in progress

when working on live-streaming loki-logs from the backend, we need a streaming-solution, where it decodes the labels-data from string to javascript-object, and does not do anything else.

this proof of concept does that.

### CODE REVIEW

File: packages/grafana-runtime/src/services/live.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ export interface StreamingFrameOptions {
   maxLength: number; // 1000
   maxDelta: number; // how long to keep things
   action: StreamingFrameAction; // default will append
+  decodeLabelsOnly?: boolean;
```
Comment (ryantxu) [inline packages/grafana-runtime/src/services/live.ts:35]: Maybe better:
```suggestion
  targetFrameType?: string;
```

And we add a frame type for "timeseries-labeled" (default is [timeseries-wide](https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/frame_type.go#L21))

Comment (gabor) [reply packages/grafana-runtime/src/services/live.ts:35]: POC at https://github.com/grafana/grafana-plugin-sdk-go/pull/483

**Comment by gabor** [comment]
dear @grafanabot keep it open please

**Comment by gabor** [comment]
keep it open please, we need to do https://github.com/grafana/grafana-plugin-sdk-go/pull/483 first.

**Comment by gabor** [comment]
keep it open pls.

**Pull request closed without merging**